docs/release-notes.md

```diff
@@ -2,51 +2,34 @@
 
 ## Resolved Issues
 
-- **Vault Deployment:** When deploying in production, customers will want to run Vault in production mode.  We have added instruction on how to configure this [here](https://open-amt-cloud-toolkit.github.io/docs/1.2/Docker/dockerLocal_prodVault/).
-- **Intel&reg; AMT connection to MPS reliability after configuration:**  We have added a small delay between when Intel&reg; AMT is activated and CIRA is configured that resolves the connection issues we were seeing in 1.1.
-
-## Known Issues in 1.3
+## Known Issues in 1.4
 #### Intel&reg; AMT
 - **Intel&reg; AMT device fails to re-connect to MPS after MPS is not available for an extended period of time:** *UPDATE: There is a firmware fix available for this issue, however, we are still testing to ensure that it completely resolves this issue.  We'll let you know once this issue is resolved.*  If the MPS goes down for more than 2 days, Intel&reg; AMT devices will no longer attempt to connect to MPS. If this happens, there are two ways to prompt Intel&reg; AMT to attempt to re-connect to MPS:
     1.	Unplug and re-plug the network cable
     2.	Reboot the Intel&reg; AMT device
 #### UI-Toolkit
 - **KVM freeze intermittently:** We have added a small delay in handling mouse interactions that prevents us from flooding the AMT channel.  There are still a few occasions where KVM could still freeze.  We are still root causing this new issue
-#### MPS
-- **AllowList:** The AllowList is a feature that is supposed to prevent devices from connecting to the MPS unless the GUID is present on the AllowList.  By default, `AllowList = false`.  If you set `AllowList = true` this won't do anything and devices will still be able to connect to MPS, even if they aren't on the AllowList.  
+#### RPS
+- **Failure to add Device:** When RPS is adding new device information to the database and the credentials to Vault, if one of these calls fail (ex. Vault not available), then RPS should revert the changes to the other service and emit an error message.  Currently RPS will provide an error, but it does not revert the changes.  Resolving the failing issue with the service and then re-running the activation will restore the data integrity between the database and Vault.
```

Comment:
**Failure to add Device:**  If an RPS call to add new device information and credentials to Vault fails, the RPS should revert the changes to the other service and emit an error message. Currently RPS will provide an error, but it does not revert the changes.  Resolving the failing issue with the service and then re-running the activation restores the data integrity between the database and Vault.

Reply:
Thank you, more good updates!